style.css

```diff
@@ -1,3 +1,226 @@
+  margin-top: 58px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--overlay-color);
+  text-align: center;
+}
+
+.menu-wrap .toggler {
+  margin-right: 58px;
+  margin-top: 36px;
+  z-index: 2;
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+  opacity: 0;
+}
+
+.menu-wrap .hamburger {
+  position: absolute;
+  top: 31px;
+  right: 38px;
+  z-index: 1;
+  width: 60px;
+  height: 60px;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.menu-wrap .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  visibility: hidden;
+  overflow: hidden;
+  display: flex;
+}
+
+.menu-wrap .menu > div {
+  background: var(--overlay-color);
+  width: 200vw;
+  height: 200vw;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  transform: scale(0);
+  padding-top: 106px;
+  transition: margin-right 0.4s ease;
+}
+
+/* Hamburger Line */
+.menu-wrap .hamburger > div {
+  position: relative;
+  flex: none;
+  width: 100%;
+  height: 2px;
+  background: var(--overlay-color);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+.menu-wrap .menu > div > div {
+  max-width: 90vw;
+  max-height: 100vh;
+  opacity: 0;
+  transition: opacity 0.4s ease;
+}
+
+.menu-wrap .menu > div > div > ul > li {
+  list-style: none;
+  color: #fff;
+  padding-bottom: 40px;
+  padding-left: 24px;
+}
+
+.menu-wrap .menu > div > div > ul > li > a {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  text-decoration: none;
+  transition: color 0.4s ease;
+  height: 24px;
+}
+
+/* Hamburger Lines - Top & Bottom */
+.menu-wrap .hamburger > div::before,
+.menu-wrap .hamburger > div::after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  top: -10px;
+  width: 100%;
+  height: 2px;
+  background: inherit;
+}
+
+/* Moves Line Down */
+.menu-wrap .hamburger > div::after {
+  top: 10px;
+}
+
+/* Turns Lines Into X */
+.menu-wrap .toggler:checked + .hamburger > div::before,
+.menu-wrap .toggler:checked + .hamburger > div::after {
+  top: 0;
+  transform: rotate(90deg);
+  background: #fff;
+}
+
+/* Toggler Animation */
+.menu-wrap .toggler:checked + .hamburger > div {
+  transform: rotate(135deg);
+  background: #fff;
+}
+
+.menu-wrap .toggler:checked ~ .menu > div {
+  transform: scale(1);
+  transition-duration: var(--menu-speed);
+}
+
+.menu-wrap .toggler:checked ~ .menu > div > div {
+  opacity: 0.9;
+  transition: opacity 0.4s ease 0.4s;
+}
+
+/* Rotate On Hover When Checked */
+.menu-wrap .toggler:checked:hover + .hamburger > div {
+  transform: rotate(225deg);
+}
+
+/* Show Menu */
+.menu-wrap .toggler:checked ~ .menu {
+  visibility: visible;
+  opacity: 0.9;
+}
+
+.pb {
+  padding-bottom: 12px;
+}
+
+.intro {
+  margin-top: 165px;
+  margin-left: 24px;
+  margin-right: 24px;
+  max-width: 327px;
+}
```

Comment:
- [ ] I think it would be better for all your sections to replicate the margins (top, bottom, left, right) defined in the design, I have highlighted some of the spaces to reduce for your reference 👇 👇 

## Your design
![image](https://user-images.githubusercontent.com/102757126/212692653-e8f83a56-973b-4e19-b233-eeb7e9d247ff.png)

## Figma design
![image](https://user-images.githubusercontent.com/102757126/212692805-5841a28b-9981-48ef-8a85-17a633939403.png)